packages/documentation/src/stories/components/forms/form-footer/form-footer.stories.ts

```diff
@@ -0,0 +1,70 @@
+import { Args, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { MetaComponent } from '@root/types';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+
+export const FooterArgs = {
+  primaryButton:
+    '<button class="btn btn-primary">Send<post-icon aria-hidden="true" name="3020"></post-icon></button>',
+  secondaryButton: '<button class="btn btn-secondary">Cancel</button>',
+  tertiaryButton:
+    '<button class="btn btn-tertiary"><post-icon aria-hidden="true" name="3024"></post-icon>Back</button>',
+};
+
+const meta: MetaComponent = {
+  id: 'f2eddf67-2c3c-40c4-bfec-df49bd028001',
+  title: 'Components/Forms/Form Footer',
+  tags: ['package:HTML'],
+  render: render,
+  parameters: {
+    badges: [],
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/JIT5AdGYqv6bDRpfBPV8XR/Foundations-%26-Components-Next-Level?node-id=1498-28215',
+    },
+  },
+  args: FooterArgs,
+  argTypes: {
+    primaryButton: {
```

Comment:
Can you separate this control into three: 
- showPrimary button
- primaryButtonType
- primaryButtonText 

Same for the other buttons, it will prevent people from testing with invalid HTML.

Reply:
I added the showXButton and xButtonText for each of them :) Do you think adding the xButtonType is necessary? As even the name suggests its type already, I don't know if we want users to be able to change the type of the button?

Reply:
You're right, it's good like this!